sentinelsat/sentinel.py

```diff
@@ -26,7 +26,6 @@
 
 class SentinelAPI:
     """Class to connect to Copernicus Open Access Hub, search and download imagery.
-
```

Comment:
Please do not remove the whitespace (here as well as in the rest of the code) that we used on purpose in the docstrings and throughout the rest of the code. I suspect this is some auto-formatting happening in your IDE. Please revert to the whitespace state of master.